api/envoy/data/cluster/v2alpha/outlier_detection_event.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package envoy.data.cluster.v2alpha;
+option java_package = "io.envoyproxy.envoy.data.cluster.v2alpha";
+option java_multiple_files = true;
```

Comment:
I think this got removed in a recent PR and is no longer being applied. Can you delete this line?